security/us-east-1/security-compliance --/README.md

```diff
@@ -22,4 +22,13 @@ AWSLOGS
 
 AWS Config snapshots are saved in a S3 Bucket of the primary region as defined in the `/security/us-east-1/security-compliance/awsconfig.tf`.
 
-In order to replicate the AWS Config bucket to a secondary region, apply the the /security/us-east-2(/security-compliance layer. The `awsconfig-replication.tf` file has a data source aiming to the AWS Config bucket in the primary region.
\ No newline at end of file
+In order to replicate the AWS Config bucket to a secondary region, apply the the /security/us-east-2(/security-compliance layer. The `awsconfig-replication.tf` file has a data source aiming to the AWS Config bucket in the primary region.
+
+
+# AWS inspector
+
+Steps to enable inspector:  
+* Switch boolean to true in the global config file (/config/common.tfvars).
+* Delegate administration to security account by running leverage terraform apply in management security-compliance layer (/management/us-east-1/security-compliance). You'd need to do this in every region you want to enable inspector.
+* Enable inspector in the security account by running leverage terraform apply in the security-compliance layer for that account, also per region (/security/us-east-1/security-compliance):
+  * Add inspector account members as needed in locals.tf, which inspector is going to enable and start monitoring.
```

Comment:
@damianleys small one here if possible let's link files with the proper hyperlink eg:  [/config/common.tfvars](../../common.tfvars) (not sure if I've correctly defined the relative path in the example, just to be seen as a reference). And for the commands let's please use `leverage terraform apply` code notation syntax for a better understanding of the README.md content. Thanks 🙏🏼